gateway/ln-gateway/src/rpc/rpc_server.rs

```diff
@@ -163,3 +165,11 @@ async fn set_configuration(
     gateway.handle_set_configuration_msg(payload).await?;
     Ok(Json(json!(())))
 }
+
+#[instrument(skip_all, err)]
+async fn is_available(
+    Extension(_): Extension<Gateway>,
+    Json(_): Json<()>,
+) -> Result<impl IntoResponse, GatewayError> {
+    Ok(Json(json!(())))
+}
```

Comment:
nit: We could probably make this an `info` endpoint that tells us a bit more about the GW, so that we e.g. know we are actually talking to the correct one.

Reply:
Changed this endpoint to `id`